File: Workshop_Senai_JavaScript/CALCULADORA.js
function CALCULATOR() 
{
    
    function OPERATION(A,B,C) 
    {
        switch (C) 
        {
    
            case 1:

                alert(`${A} + ${B} = ${A + B}`)
                return A + B
    
            case 2:
                
                alert(`${A} - ${B} = ${A - B}`)
                return A - B
    
            case 3:

                alert(`${A} × ${B} = ${A * B}`)
                return A * B
    
            case 4:

                alert(`${A} ÷ ${B} = ${A / B}`)
                return A / B
    
            default:
    
                alert('OPERADOR INVÁLIDO!');
                CALCULATOR()
                break;
    
        }

    }

    function CONTINUE() 
    {

        var PROCEED = Number(prompt("(1)CONTINUAR \n(2)RESETAR \n(3)SAIR"))

        switch (PROCEED) 
        {
            case 1:

                N1 = RESULT
                var OPERATOR = Number(prompt("QUAL OPERAÇÃO DESEJA EFETUAR: \n(1)SOMA \n(2)SUBTRAÇÃO \n(3)MULTIPLICAÇÃO \n(4)DIVISÃO"))
                var N2 = Number(prompt("DIGITE O SEGUNDO NÚMERO:"))
                RESULT = OPERATION(N1,N2,OPERATOR)
                CONTINUE()
                break;
    
            case 2:

                CALCULATOR()
                break;
    
            case 3:

                alert("TUDO BEM, ATÉ MAIS!")
                break;
    
            default:

                break;
                
        }
        
    }

    var N1 = Number(prompt("DIGITE O PRIMEIRO NÚMERO:"))
    var OPERATOR = Number(prompt("QUAL OPERAÇÃO DESEJA EFETUAR: \n(1)SOMA \n(2)SUBTRAÇÃO \n(3)MULTIPLICAÇÃO \n(4)DIVISÃO"))
    var N2 = Number(prompt("DIGITE O SEGUNDO NÚMERO:"))
    var RESULT = OPERATION(N1,N2,OPERATOR)
    CONTINUE()

}
console.clear()
CALCULATOR()

File: Workshop_Senai_C++/CALCULADORA.cpp
#include <iostream>
using namespace std;

int loop();

int main(){
    int OP;
    float N1, N2;
    
    cout << "Olá, bem vindo a calculadora feita em C++!" << endl;
    
    cout << "Qual o primeiro número?" << endl;
    cin >> N1;

    cout << "Qual operador você irá usar? 1- Somar // 2- Subtrair // 3- Multiplicar // 4- Dividir" << endl;
    cin >> OP;

    switch(OP){
        case 1: //Soma
            cout << "Qual o segundo número?" << endl;
            cin >> N2;

            cout << "O resultado da soma é: " << N1 + N2 << endl<< endl;
            
        break;

	    case 2: //Subtração
            cout << "Qual o segundo número?" << endl;
            cin >> N2;

            cout << "O resultado da subtração é: " << N1 - N2 << endl;
            
	    break;
	    
	    case 3: //Multiplicação
            cout << "Qual o segundo número?" << endl;
            cin >> N2;

            cout << "O resultado da multiplicação é: " << N1 * N2 << endl<< endl;
            
        break;
        
        case 4: //Divisão
            cout << "Qual o segundo número?" << endl;
            cin >> N2;

            cout << "O resultado da divisão é: " << N1 / N2 << endl<< endl;
            
        break;
    }
	
    loop();
	
}

int loop(){
    int DESEJO;
    
    cout << "Você deseja: 1- Parar por aqui // 2- Fazer outro cálculo" << endl; //Reutilizar ou não
    cin >> DESEJO;
            
    switch(DESEJO){
        case 1:
		    cout << "Tudo bem, até mais!";
        break;
                
        case 2:
            main();
        break;
    }
    return DESEJO;
}
